src/Goldilocks/Program.cs

```diff
@@ -1,43 +1,29 @@
 using System.Text.Json.Serialization;
+using Goldilocks;
 
 var builder = WebApplication.CreateBuilder(args);
+builder.Logging.AddConsole();
```

Comment:
Let's remove this line for the benchmarks.

Reply:
I wonder if that is "fair" to report, both for app size and RPS/perf.

Reply:
None of our other benchmarks have logging enabled. We could certainly capture both sets of figures if we wish to track the impact of logging, but the default logging levels include multiple messages per request which will significantly impact the RPS results.

Reply:
In order **not** to have logging enabled, a line actually needs to be added:

```c#
builder.Logging.ClearProviders();
```

Reply:
Yep, until the new "slim" builder is available.